src/Tribe/Main.php

```diff
@@ -371,6 +371,9 @@ public function bind_implementations() {
 
 		// Views V2
 		tribe_register_provider( Tribe\Tickets\Events\Views\V2\Service_Provider::class );
+
+		// RSVP
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 420b889634ca48c156254152dcb54d9f</sub>